### DISCUSSION TITLE
<Export all my Stars>

### DISCUSSION THREAD

**Discussion by jbionic2016** [community/community#65045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there,

I wonder how I can export the complete list of my Stars (bookmarked projects) in a simple way without having to install anything?

Thanks in advance.

**Comment by nvspavankalyanch** [comment]
Hi there,

Exporting your list of starred projects from GitHub is indeed a useful task. While GitHub's interface doesn't provide a direct export feature for starred repositories, you can achieve this through a few steps:

**1. GitHub API**: You can use the GitHub API to retrieve a list of your starred repositories. You don't need to install anything; you can make API requests using tools like curl or browser-based REST clients like Postman or Insomnia.

To get the list of your starred repositories, you can make a GET request to the following endpoint:

`GET /user/starred`

Replace "user" with your GitHub username. You'll need to authenticate using your personal access token for API requests.

**2. GitHub CLI (Command Line Interface)**: If you're comfortable using the command line, you can install and use GitHub CLI. It provides a simple command to list your starred repositories.

Install GitHub CLI: https://cli.github.com/

Once installed, you can use the following command:

`gh repo list --starred`

**Remember to replace "user" with your GitHub username.**

These methods should help you export your list of starred projects without having to install anything substantial. If you need further assistance, feel free to ask!

**Reply by jbionic2016** [reply]
I think it should be easier than that via click & download button

**Reply by nvspavankalyanch** [reply]
Absolutely, having a "click & download" option would indeed be fantastic. Regrettably, at the present moment, such a feature doesn't exist.. It's quite possible that they could consider incorporating such a feature to enhance user experience in upcoming releases. Let's remain hopeful for these exciting possibilities in the future!

**Reply by 4ff0rdsUn1c0rns** [reply]
The UX includes CLI etc in a community like this I presume; incentives to learn and apply and automate.

**Reply by BasixKOR** [reply]
The CLI doesn't have a flag called `--starred`. Please stop polluting the community with AI slops.

**Comment by 4ff0rdsUn1c0rns** [comment]
Does this separate lists?
